kong.conf.default

```diff
@@ -1989,3 +1989,105 @@
+#    block, allowing various Wasmtime-specific flags to be set.
+#
+# The documentation for all supported directives can be found in the Nginx Wasm
+# module repository:
+#
+# https://github.com/Kong/ngx_wasm_module/blob/main/docs/DIRECTIVES.md
+#
+# Additionally, there are several Wasm-specific directives which can be tuned
+# specifically at the `http {}` or `server {}` level. These are supported via
+# the `nginx_http_<directive>` and `nginx_proxy_<directive>` namespaces:
+#
+# - `http {}`
+#   * `nginx_http_wasm_socket_connect_timeout`
+#   * `nginx_http_wasm_socket_read_timeout`
+#   * `nginx_http_wasm_socket_send_timeout`
+#   * `nginx_http_wasm_socket_buffer_size`
+# - `server {}`
+#   * `nginx_proxy_wasm_socket_connect_timeout`
+#   * `nginx_proxy_wasm_socket_read_timeout`
+#   * `nginx_proxy_wasm_socket_send_timeout`
+#   * `nginx_proxy_wasm_socket_buffer_size`
+#
+# The values for these settings are inherited from their `nginx_*_lua_*`
+# counterparts if they have not been explicitly set. For instance, if you set
+# `nginx_http_lua_socket_connect_timeout` in your `kong.conf` file, the value
+# of this setting will be propagated to `nginx_http_wasm_socket_connect_timeout`
+# unless you _also_ set `nginx_http_wasm_socket_connect_timeout`.
+#
+# Some TLS-related settings receive special treatment as well:
+#
+# - `lua_ssl_trusted_certificate`: when set, the value is  propagated to the
+#   `nginx_wasm_tls_trusted_certificate` directive.
+# - `lua_ssl_verify_depth`: when set (to a value greater than zero), several
+#   TLS-related `nginx_wasm_*` settings are enabled:
+#   * `nginx_wasm_tls_verify_cert`
+#   * `nginx_wasm_tls_verify_host`
+#   * `nginx_wasm_tls_no_verify_warn`
+#
+# Like other `kong.conf` fields, all injected Nginx directives documented here
+# can be set via environment variable. For instance, setting:
+#   `KONG_NGINX_WASM_TLS_VERIFY_CERT=<value>`
+#
+#   Will inject the following in to the `wasm {}` block:
+#
+#   `tls_verify_cert <value>;`
+#
+#
+# There are several Nginx directives supported by `ngx_wasm_module` which should
+# not be used because they are irrelevant to or unsupported by Kong, or they may
+# conflict with Kong's own management of Proxy-Wasm. Use of these directives may
+# result in unintentional breakage:
+#
+# - `wasm_call`
+#   * `nginx_http_wasm_call`
+#   * `nginx_proxy_wasm_call`
+# - `module`
+#   * `nginx_wasm_module`
+# - `proxy_wasm`
+#   * `nginx_http_proxy_wasm`
+#   * `nginx_proxy_proxy_wasm`
+# - `proxy_wasm_isolation`
+#   * `nginx_http_proxy_wasm_isolation`
+#   * `nginx_proxy_proxy_wasm_isolation`
+# - `resolver_add`
+#   * `nginx_wasm_resolver_add`
+#   * `nginx_http_resolver_add`
+#   * `nginx_proxy_resolver_add`
+# - `proxy_wasm_request_headers_in_access`
+#   * `nginx_http_proxy_wasm_request_headers_in_access`
+#   * `nginx_proxy_proxy_wasm_request_headers_in_access`
+# - `shm_queue`
+#   * `nginx_wasm_shm_queue`
```

Comment:
Is this too verbose? Should I revert it to just the "normal" directive names (e.g. `wasm_call`) and not include the fully-qualified/kong-injected names (e.g. `nginx_http_wasm_call`)?